arithmetization/src/test/java/net/consensys/linea/zktracer/ReplayTests.java

```diff
@@ -200,54 +206,54 @@ void fatMxp() {
   @Test
   void bulkReplay() {
     // bulkReplay("./src/test/resources/replays");
```

Comment:
Can we also update this comment to inlcude LINEA_MAINNET?
// bulkReplay(LINEA_MAINNET, "./src/test/resources/replays");

Reply:
Yes, we can do.  Create a separate issue and I will add support there.  I haven't completed grokked the `builkReplay` system, so I need to talk it over with you first, etc.

Reply:
Thank you 👍

Reply:
It would be good also to merge the two replay methods somehow.  I will have a look at that also